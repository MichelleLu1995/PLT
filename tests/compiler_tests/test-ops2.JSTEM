def int main()
{
  printb(True);
  printb(False);
  printb(True && True);
  printb(True && False);
  printb(False && True);
  printb(False && False);
  printb(True || True);
  printb(True || False);
  printb(False || True);
  printb(False || False);
  printb(!False);
  printb(!True);
  print(-10);
  print(--42);
}