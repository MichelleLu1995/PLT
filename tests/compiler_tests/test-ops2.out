101000111010-10
